Add VM based install, adapt retail to SUMA5 (#3046)
uyuni-project/uyuni-docs: modules/retail/pages/retail-install-setup.adoc

[IMPORTANT]
====
POS image templates and resulting images do not set a system user password.
You will not be able to login as a system user to a system that has been installed with a {suse} provided template without customization.
However you can use Salt to manage clients without a system user password.

> You can use Salt to setup a system user password after the terminal has been deployed.

Reviewer: ```suggestion
You can use {salt} to set up a system user password after the terminal has been deployed.
```

====

=== Validate POS Image Registration

After POS image is successfully built and inspected by {productname}, you can validate that POS boot image was correctly registered by following procedure.

.Procedure Validate POS boot image

> . On {productname} command line execute following command

Reviewer: ```suggestion
. On {productname} command line execute following command:
```

[IMPORTANT]
====
{productname} {smr} branch proxy uses different [systemitem]``filename`` then previous non-containerized branch server.

For containerized branch proxy set [systemitem]``filename`` to the [systemitem]``pxelinux.0`` for BIOS based system and [systemitem]``grub/shim.efi`` for UEFI systems with SecureBoot.
====

== Terminal Partitioning and Image Selection

{saltboot} requires instructions how to partitioning terminal harddisk and what image to deploy.
This is done individually for each hardware type of terminals.
For more information about hardware types and partitioning, see xref:retail:retail-deploy-terminals.adoc[].

Above mentioned steps are mandatory minimum for successful {saltboot} deployment.
For configuration examples, see xref:retail:example-configurations.adoc[].

== Synchronize Images to the Branch Server

{productname} {smr} {productnumber} no longer need manual image synchronization, all images are available to all clients automatically.

> This may not be always desired, for example to allow images gradual deployments across all branches. For a way how to limit image deployment, see xref:retail:retail-best-practices.adoc[].

Reviewer: ```suggestion
This may not be always desired, for example to allow images gradual deployments across all branches.
For a way how to limit image deployment, see xref:retail:retail-best-practices.adoc[].
```
